corehq/messaging/scheduling/static/scheduling/js/conditional_alert_main.js

```diff
@@ -34,5 +34,34 @@ hqDefine("scheduling/js/conditional_alert_main", [
         // setup tab
         basicInformation.setRuleTabVisibility();
         $("#conditional-alert-basic-info-panel").koApplyBindings(basicInformation);
+
+        $("#conditional-alert-save-btn").click(function () {
+            var newRuleName = $("#id_conditional-alert-name").val();
+            if (!name) {
+                $("#conditional-alert-form").submit();
+            } else if (name !== newRuleName) {
+                var caseType = $(
```

Comment:
You can get this value with `$("#id_criteria-case_type").val()` - it's better to deal with the underlying HTML element than to depend on implementation details of select2. That gives us the flexibility to change the UI, say, to switch widgets, without needing to change this code.